ios/Classes/TiAirshipModule.m

```diff
@@ -118,6 +118,28 @@ - (void)setUserNotificationsEnabled:(id)value {
     [[UAirship push] updateRegistration];
 }
 
+- (id)isAutoBadgeEnabled {
+    return NUMBOOL([UAirship push].autobadgeEnabled);
+}
+
+- (void)setIsAutoBadgeEnabled:(id)args {
+    [UAirship push].autobadgeEnabled = [TiUtils boolValue:args
+                                                      def:[UAirship push].autobadgeEnabled];
+    [[UAirship push] updateRegistration];
+}
+
+- (id)badgeNumber {
+    return NUMINTEGER([UAirship push].badgeNumber);
+}
+
+- (void)setBadgeNumber:(id)args {
+    [UAirship push].badgeNumber = [TiUtils numberFromObject:args].integerValue;
```

Comment:
Does this need a default value? Or likewise could the other one be done with:

`[TiUtils numberFromObject:args].boolValue`